package.json

```diff
@@ -11,11 +11,13 @@
   ],
   "dependencies": {},
   "devDependencies": {
```

Comment:
I'm trying to be careful. I am not sure that bundling es5-shim is a good experience for downstream developers. (I may just need enlightenment on that point)